src/utils/apis/LoginAPI.js

```diff
@@ -12,6 +19,10 @@ export const loginAPI = {
                 mail: login,
                 password: password
             })
+        }).then(response => {
+            jwtUtils.saveTokenFromResponse(response)
+            setInterval(jwtUtils.refreshToken, 2000)
```

Comment:
2 sekundy to chyba trochę za szybko - sprawdź na backu, ile jest ważny i w jakiejś połowie czasu odświeżaj

Reply:
2 sekundy tak dałem żeby można było sobie zobaczyć że to działa git, oczywiście w normalnie działającym systemie to do zupy taki krótki czas i to zmienię

Reply:
Zastanawiam się, czy nie lepiej byłoby go ustawić przez envy - wtedy nie trzeba będzie robić jeszcze raz deploy'a, jak się zmieni - to samo przy Recruitment Service